topics/transcriptomics/tutorials/scrna-raceid/tutorial.md

```diff
@@ -0,0 +1,690 @@
+>                - *"List of clusters"*: `1`
+>        - In *"Cells in Set B"*:
+>            - *"Name of Set"*: `Cells in 3`
+>            - *"Selection method"*: `Cluster Numbers`
+>                - *"List of clusters"*: `3`
+>        - *"Use Defaults?"*: `Yes`
+>
+{: .hands_on}
+
+![Cluster Inspection of Cells]({{site.baseurl}}{% link topics/transcriptomics/images/raceid_clustinspect_cells.png %} "MA plot of cells in cluster 1 and cluster 3")
+
+The genes shown as grey dots are not labelled because they are of similar levels of gene expression variability in both clusters, but the genes as labelled red dots on the fringes do display significant changes between the clusters.
+
+> ### {% icon question %} Questions
+>
+> Is *Gstm3* (at position 1.5, -2.8 on the MA plot) more significantly differentially expressed than *Ptma* (at position 3.6, 1.2)?
+>
+> > ### {% icon solution %} Solution
+> >
+> > *Gstm3* has a lower overall total expression in both clusters than *Ptma* which has significantly more total expression, but *Gstm3* is primarily expressed in `c1` cells and *Ptma* more in `c3` cells, showing a clear difference in their expression profiles.
+> >
+> {: .solution}
+>
+{: .question}
+
+
+## Differential Gene Expression Across All Clusters
+
+We will now look at some genes of interest to see how prevalent or unique they are across clusters. Usually known marker genes are used to identify clusters by their cell type and not just a number, but any gene of interest can be used if it is believed to characterise a cluster of cells.
+
+Here we will look at the combined expression of *Gstm3*, *St3gal4*, and *Gna11* which all had adjusted P-values of less than 1e-15 in cluster 1.
+
+> ### {% icon hands_on %} Hands-on: Task description
+>
+> 1. **Cluster Inspection using RaceID** {% icon tool %} with the following parameters:
+>    - {% icon param-file %} *"Input RaceID RDS"*: `outrdat` (output of **Clustering using RaceID** {% icon tool %})
+>    - *"Plot All Clusters?"*: `No`
+>    - *"Perform Subset Analysis?"*: `No`
+>    - *"Examine Genes of Interest"*: `Yes`
+>        - *"Genes to Examine"*: `Gstm3, St3gal4, Gna11`
+>        - *"Use Defaults?"*: `Yes`
+>    - *"Differential Gene Testing"*: `No`
+>
+{: .hands_on}
+
+
+![Expression Plot]({{site.baseurl}}{% link topics/transcriptomics/images/raceid_goi.png %} "Expression plot of genes of interest across different cells.")
+
+
+
+> ### {% icon question %} Question
+>
+> Are these genes expressed where we expect?
+>
+> > ### {% icon solution %} Solution
+> >
+> > They appear to overlap `c6` which is in close proximity to `c1`. There are two reasons why this might be the case:
+> > 1. `c1` is a small cluster and noisy cluster surrounded by more stably defined neighbours.
+> > 2. The three genes are more differentially expressed in `c1` than in `c6` if we look at them as independent entities, but their combined effect tells us that together they describe `c6` better.
```

Comment:
I do not think it's clear where you see c1 and c6 here ... the user needs to relate this to the images above ... maybe mention this in the image description and maybe also mention the scratchbook feature of galaxy to put both images next to each other.